op-bindings/bindings/gaspriceoracle.go

```diff
@@ -30,8 +30,8 @@ var (
 
 // GasPriceOracleMetaData contains all meta data concerning the GasPriceOracle contract.
```

Comment:
just fyi, the `op-bindings` package is on its way out, once this PR is merged (and nothing major in flight that would have merge conflicts) then I will `mv op-bindings/bindings op-e2e/bindings`. It is now the responsibility of services to maintain up to date bindings

Reply:
Thanks 🙏